inspire_json_merger/config.py

```diff
@@ -317,3 +318,20 @@ class GrobidOnArxivAuthorsOperations(MergerConfigurationOperations):
     }
     comparators = GROBID_ON_ARXIV_COMPARATORS
     conflict_filters = ["authors.full_name"]
+
+
+class ErratumOnPublisherOperations(MergerConfigurationOperations):
+    pre_filters = [
+        update_material,
+        filter_curated_references,
+        update_authors_with_ordering_info,
+        clean_root_for_acquisition_source
+    ]
+    default_list_merge_op = U.KEEP_ONLY_HEAD_ENTITIES
```

Comment:
After some more thought, I think we need to do it the other way around, namely make the default here `KEEP_UPDATE_AND_HEAD_ENTITIES_HEAD_FIRST`, then configure `abstracts` with `KEEP_ONLY_HEAD_ENTITIES`. It's not an issue if we add a new (hidden) title with the erratum.